handlers/gallery.go

```diff
@@ -1,7 +1,60 @@
 package handlers
 
-import "net/http"
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"path/filepath"
+)
+
+type JSONInfo struct {
+	Content string
+}
+
+type JSONPhoto struct {
+	ID                  string
+	Primary             string
+	Secret              string
+	Server              string
+	Farm                int
+	Photos              string
+	Videos              int
+	Title               JSONInfo
+	Description         JSONInfo
+	NeedsInterstitial   int
+	VisibilityCanSeeSet int
+	CountViews          string
+	CountComments       string
+	CanComment          int
+	CateCreate          string
+	CateUpdate          string
+}
+
+type JSONObject struct {
+	Photosets struct {
+		Page     int
+		Pages    int
+		Perpage  int
+		Potal    int
+		Photoset []JSONPhoto
+	}
+}
+
+var albums JSONObject
```

Comment:
Please create a galleryService (use [eventService](https://github.com/hacksoc-manchester/www/blob/master/services/eventService/eventService.go) as inspiration) and move all of these operations there.